tendrl/monitoring_integration/grafana/datasource.py

```diff
@@ -0,0 +1,45 @@
+import __builtin__
+import json
+import traceback
+from requests import post
+from grafana.connection import port_open
+from grafana.exceptions import ConnectionFailedException
+
+HEADERS = {"Accept": "application/json",
+           "Content-Type": "application/json"
+           }
+
+
+''' Create Datasource '''
+
+
+def post_datasource(datasource_json):
```

Comment:
post_datasource is just a call to create a new datasource used by create_datasource

I have tried to make it modular thats why I have split it.